content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,302 @@
+*North America*.
+
+.. image:: lead_distribution_report/sales-team-location.png
+  :align: center
+  :alt: Use Sales Team to filter the location the lead is associated with.
+
+By adding this new rule, a new line appears at the top of the pop-up menu showing :guilabel:`Match
+any of:` with a :icon:`fa-caret-down` :guilabel:`(down caret)` icon.
+
+Click the icon, then select :guilabel:`all` from the resulting list. This setting causes the filter
+to **only** show records that match all the rules contained inside. This uses the same logic as an
+AND (&) logical operator.
+
+.. _phone-number:
+
+3. Contact method
+~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, select :guilabel:`is set` from the drop-down menu in the second field.
+Selecting this operator **only** filters for records that have a phone number associated with the
+lead.
+
+Click :guilabel:`New rule` and set the first field to :guilabel:`Email`. Then, select :guilabel:`is
+set` from the drop-down menu in the second field.
+
+These rules add only leads with an associated contact method to the report.
+
+.. _active-status:
+
+1. Active status
+~~~~~~~~~~~~~~~~
+
+Click the :icon:`fa-sitemap` :guilabel:`(Branch)` icon to the right of the `Phone is set` line, to
+add a new rule that branches from the rules above.
+
+Two horizontal sets of fields appear below a line showing :guilabel:`any of:` with a
+:icon:`fa-caret-down` :guilabel:`(down caret)` icon. This setting filters for records that match
+**any** of the rules contained inside. This uses the same logic as an OR (|) logical operator.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+Next, click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+
+.. image:: lead_distribution_report/active-set.png
+  :align: center
+  :alt: Use Active to include active status in the report.
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`essential-filters` in the order specified to achieve a heavily-detailed filter.
+
+:ref:`referred-by`
+
+:ref:`source`
+
+:ref:`notes`
+
+:ref:`tags`
+
+:ref:`email`
+
+:ref:`salesperson`
```

Comment:
I see what you did here however I disagree with the presentation and most of the information in the corresponding sections.

In order to convey these extras quickly, a bullet list with brief list item descriptions for each value would suffice. At the end can add a **see also** pointing to the [additional rules for quality leads](https://www.odoo.com/documentation/17.0/applications/sales/crm/track_leads/quality_leads_report.html#add-rules-for-quality-leads) which covers those values adequately in more detail.